hdf5-sys/src/h5es.rs

```diff
@@ -0,0 +1,84 @@
+//! Event set module
+use crate::internal_prelude::*;
+
+#[repr(C)]
+pub struct H5ES_err_info_t {
+    api_cname: *mut c_char,
+    api_args: *mut c_char,
+
+    app_file_name: *mut c_char,
+    app_func_name: *mut c_char,
+    app_line_num: c_uint,
+
+    op_ins_count: u64,
+    op_ins_ts: u64,
+    op_exec_ts: u64,
+    op_exec_time: u64,
+
+    err_stack_id: hid_t,
+}
+
+#[repr(C)]
+pub struct H5ES_op_info_t {
+    api_cname: *const c_char,
+    api_args: *mut c_char,
+
+    app_file_name: *const c_char,
+    app_func_name: *const c_char,
+    app_line_num: c_uint,
+
+    op_ins_count: u64,
+    op_ins_ts: u64,
+    op_exec_ts: u64,
+    op_exec_time: u64,
+}
+
+#[repr(C)]
+pub enum H5ES_status_t {
```

Comment:
For enums:
- Common derives?
- Specify values explicitly?

Reply:
Derives added, but values are not specified since they are not specified in `hdf5-c`

Reply:
Fair enough, my bad, I guess I haven't dealt with C enums in a long while; just checked - the generated size by default is 4 (i.e. same as `c_int`) and the first value is 0, so it will try to mimic C behaviour on the platform.